adventure/api.py

```diff
@@ -63,5 +63,11 @@ def move(request):
 @csrf_exempt
 @api_view(["POST"])
 def say(request):
-    # IMPLEMENT
-    return JsonResponse({'error':"Not yet implemented"}, safe=True, status=500)
+    player = request.user.player
+    room = player.room()
+    currentPlayerUUIDs = room.playerUUIDs(player.id)
```

Comment:
This is a great start on this! I think you might want to add something for player_id and some other data but this structure should work.